### PR TITLE
[ENH] Agilent read interferogram data (seq and drd)

### LINKED ISSUE #243: Read .seq Agilent file
We need to read .seq files to do FFT

### PR DESCRIPTION
**Pull request by stuart-cls**
Add support for reading agilent interferogram data files, including metadata required to calculate FFT properly (See #261)

Closes #243

### CODE REVIEW

File: orangecontrib/spectroscopy/agilent.py [hunk 1]
```diff
@@ -98,8 +98,9 @@ def _get_prop_d(dat, param):
         return val
 
     def _get_prop_str(dat, param):
```
Comment (markotoplak) [inline orangecontrib/spectroscopy/agilent.py]: I would have named this internal helpers differently: I would not use the underscore. Also, "get" can often seems a bit superfluous.

File: orangecontrib/spectroscopy/agilent.py [hunk 2]
```diff
@@ -245,62 +353,186 @@ class agilentMosaic(DataObject):
     https://bitbucket.org/AlexHenderson/agilent-file-formats
     """
 
+    def __init__(self, filename, MAT=False):
+        super().__init__(filename, MAT)
+        self._get_data()
+
+    def _get_data(self):
```
Comment (markotoplak) [inline orangecontrib/spectroscopy/agilent.py]: For, the name _get_data was a bit confusing and was looking for what it actually returns. Perhaps a name such as _prepare_data or _load_data or similar would be more appropriate next time.

**Comment by markotoplak** [comment]
I like that you are handling tiled images... Let's see where this gets to.

**Comment by stuart-cls** [comment]
Thanks Marko, I agree the internal method naming is not the best in the agilent module. I'll make a note on that.

If you would like to try the tile-by-tile file loader, it is the only widget in the [prototypes](https://github.com/Quasars/orange-spectroscopy-prototypes) repo right now. There is an example .ows included showing how to use it. Some of our users have been putting it through its paces for a few months now without issue.

**Pull request merged**